src/api/tigl.h

```diff
@@ -3126,8 +3125,44 @@ TIGL_COMMON_EXPORT TiglReturnCode tiglIntersectWithPlaneSegment(TiglCPACSConfigu
                                                                 char** intersectionID);
 
 /**
-* @brief tiglIntersectGetLineCount return the number of intersection lines computed by 
-* ::tiglIntersectComponents or ::tiglIntersectWithPlane for the given intersectionID.
+* @brief tiglGetCurveIntersection computes intersection point of two (intersection) curves.
+* Both curves are the result of a previous intersection (e.g. Fuselage-Wing Intersection).
+*
+* The curves must be specified by their intersectionIDs and an index of the wire in the
+* intersection. The intersection is calculated to a specified tolerance.
```

Comment:
I think we should explain in more detail, what tolerance is. Either describe in words (e.g. distance between the curves) or with an image,